contrib/pipelines/secadvisor/README.md

```diff
@@ -5,30 +5,104 @@ Implements the IBM Security Advisor pipeline
 
 ## Setup Skydive
 
-First build and run skydive:
+Follow instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the skydive code.
+Build and run skydive:
 
 ```
 make
 cp etc/skydive.yml.default skydive.yml
-sudo $(which skydive)) allineone -c skydive.yml
+// adjust settings in skydive.yml
+sudo $(which skydive) allineone -c skydive.yml
 ```
 
+Alternatively, run a skydive agent on each of your hosts and a skydive analyzer on one host.
+
+On each machine:
+
+```
+sudo skydive agent -c skydive.yml
+```
+
+Be sure to set the field "analyzers" in the skydove.yml to point to the analyzer.
+
+On one machine:
+
+```
+sudo skydive analyzer -c skydive.yml
+```
+
+
 ## Setup Minio
 
-Install and run minio (an AWS S3-like object store).
+Follow the instructions on https://docs.min.io/docs/minio-quickstart-guide.html to install and run a minio object store (an AWS S3-like object store).
+
+```
+wget https://dl.min.io/server/minio/release/linux-amd64/minio
+chmod +x minio
+./minio server /data
+```
+
+This will run a basic minio object store server on your machine, placing the objects in "/data".
+Note the printed output in the logs from the ./minio command and take down the values of "Endpoint", "AccessKey", and "SecretKey". Place these items in the secadvisor.yml in the appropriate locations (see below). 
+Create a bucket in the object store to hold the security-advisor data.
+
+## Prepare secadvisor.yml
+
+```
+cp secadvisor.yml.default secadvisor.yml
+```
+
+Set the fields "endpoint", "access_key", "secret_key", "bucket", "cluster_net_masks", etc.
+The subnets specified in cluster_net_masks are used to determine whether a flow is internal, ingress, or egress. Enter in the list all of the subnets that are recognized as being inside your cluster.
+The types of "excluded_tags" that are recognized are: internal, ingress, egress, other.
 
 ## Setup Pipeline
 
-Build and run the pipeline:
+Build and run the pipeline in the secadvisor directory:
 
 ```
 make all
 make install
 make run
 ```
 
+or
+
+```
+secadvisor secadvisor.yml
+```
+
 ## Generate Flows
 
 Via the Skydive WebUI setup captures and generate traffic which should
 result in the secadvisor pipeline sending flows to the ObjectStore.
 
+For example, run some iperf traffic between entities and capture the flow via the Skydive GUI.
+Objects are created in the Object Store only if there is flow information to be saved.
+Check your bucket in Object Store and see a new object about once per minute.
```

Comment:
(1) I propose to add a screen show 
(2) If there is any programmatic way, I would advice to also show that